io.home-assistant.gas/manifest.tt

```diff
@@ -0,0 +1,13 @@
+abstract class @io.home-assistant.gas
+#_[thingpedia_name="Gas Binary Sensor"]
+#_[thingpedia_description="Interface for Home Assistant's Gas Binary Sensor."]
+#_[description="Interface for Home Assistant's Gas Binary Sensor."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out state : Enum(detecting_gas,not_detecting_gas), out value : String)
```

Comment:
The word gas is probably redundant in the enum value, given this is a gas sensor (and if you don't have it, you get slightly better generalization as the enum is merged with other similar enums).
Also, what's value and why is it a string? Shouldn't it be a number instead?